src/control/server/storage/scm/ipmctl_test.go

```diff
@@ -73,12 +73,23 @@ func MockModule(d *ipmctl.DeviceDiscovery) storage.ScmModule {
 type mockIpmctl struct {
 	discoverModulesRet error
 	modules            []ipmctl.DeviceDiscovery
+	getFWInfoRet       error
+	fwInfo             ipmctl.DeviceFirmwareInfo
+	updateFirmwareRet  error
```

Comment:
Rather than adding more fields to the mock implementation, would prefer to convert this over to using a `mockIpmctlCfg` struct similar to how other mocks work.

Reply:
Good catch, converting